packages/connectivity/src/scp-cf/destination/destination-cache.ts

```diff
@@ -49,35 +109,39 @@ export interface DestinationCacheType {
  * @returns A destination cache object.
  * @internal
  */
-export const DestinationCache = (
-  cache: Cache<Destination>
-): DestinationCacheType => ({
-  retrieveDestinationFromCache: (
+export const DestinationCache = <T extends DestinationCacheInterface>(
+  cache: T = new DefaultDestinationCache({
+    hours: 0,
+    minutes: 5,
+    seconds: 0
+  }) as any
```

Comment:
[req] The `as any` should not be needed anymore

Reply:
Besides that looks good to me.

Reply:
Without `any` it gives an error:

`'DefaultDestinationCache' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint 'DestinationCacheInterface'.`